roles/foreman_provisioning/tasks/main.yml

```diff
@@ -13,7 +13,7 @@
     - Ubuntu mirror
   shell: >
     {{ foreman_provisioning_hammer }} medium update --name "{{ item }}" --organization-titles "{{ foreman_provisioning_organization }}" --location-titles "{{ foreman_provisioning_location }}"
-  when: foreman_provisioning_use_taxonomies and foreman_provisioning_foreman_version == 'nightly'
+  when: foreman_provisioning_foreman_version == 'nightly'
```

Comment:
Why do you only do this on nightly? Is this even needed now that we have seeding scripts that should always assign it to a taxonomy?

Reply:
Not needed now, seeds take care of it. Good catch.